### PR TITLE
FirstLauncAppHelper and UserDetailsCountlyService exception thrown fixed

### PR DESCRIPTION
**Pull request by AliRKat**

### CODE REVIEW

File: Assets/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public async Task SetUserDetailsAsync(CountlyUserDetailsModel userDetailsModel)
                 }
 
                 if (!_countlyUtils.IsPictureValid(userDetailsModel.PictureUrl)) {
-                    throw new Exception("Accepted picture formats are .png, .gif and .jpeg");
+                    Log.Warning("Accepted picture formats are .png, .gif and .jpeg");
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Services/UserDetailsCountlyService.cs:71]: Better log message

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
modify the getter to have a implicit "process" call and print a warning

Add a null check for picture path handling

Add a test for when the picture path is null and empty and verify that the serialization is appropriate

let's add some comments in the test for "sections"

rework the test comments in the test

**Review by ArtursKadikis — APPROVED** [review]
